src/IssueViz.Security.UnitTests/HotspotsList/HotspotsControlTests.cs

```diff
@@ -45,6 +46,18 @@ public void Ctor_HotspotsListIsWpfTableControl()
             testSubject.hotspotsList.Child.Should().Be(uiControl);
         }
 
+        [TestMethod]
```

Comment:
I had tests for the navigation as well but had to take them out.. left this one though. It's unrelated to the PR.